pkgs/os-specific/darwin/apple-source-releases/default.nix

```diff
@@ -146,7 +146,7 @@ let
       then stdenv.fetchurlBoot
       else fetchurl;
   in fetch {
-    url = "http://www.opensource.apple.com/tarballs/${pname}/${pname}-${version}.tar.gz";
+    url = "https://github.com/apple-oss-distributions/${pname}/archive/refs/tags/${pname}-${version}.tar.gz";
```

Comment:
and probably here, too

Reply:
I've fixed the others, but this one seems tricky.